apps/admin/frontend/src/app_write_in_flows.test.tsx

```diff
@@ -262,97 +252,4 @@ test('manual write-in data end-to-end test', async () => {
   within(zooCouncilMammal2).getByText(hasTextAcrossElements('Chimera10'));
   within(zooCouncilMammal2).getByText(hasTextAcrossElements('Rapidash20'));
   within(zooCouncilFish2).getByText(hasTextAcrossElements('Relicanth14'));
-
-  userEvent.click(screen.getByText('Reports'));
-});
-
-test('availability of write-in tally report', async () => {
```

Comment:
moved this testing to more thorough testing in `tally_writein_report_screen.test.tsx`.